sql/seanCollection_2019-09-23.sql

```diff
@@ -0,0 +1,57 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.7.27)
+# Database: seanCollection
+# Generation Time: 2019-09-23 13:29:12 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table things
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `things`;
+
+CREATE TABLE `things` (
+  `id` int(1) unsigned NOT NULL AUTO_INCREMENT,
+  `widgetName` varchar(255) DEFAULT NULL,
+  `widgetDescription` varchar(255) DEFAULT NULL,
+  `widgetSize` int(11) DEFAULT NULL,
+  `widgetRating` int(11) DEFAULT NULL,
```

Comment:
If they're all widgets, you can probably get away with just 'name', 'description' etc instead of adding 'widget' to everything

Reply:
Charlie told me to untick the 'allow Null' boxes otherwise you'll be able to add data to the app with some of the fields left blank

Reply:
> If they're all widgets, you can probably get away with just 'name', 'description' etc instead of adding 'widget' to everything
Common Practice so as not to get confused with data from other tables. Admittedly not needed as we are only working with a single table. But there you go.

Reply:
> > If they're all widgets, you can probably get away with just 'name', 'description' etc instead of adding 'widget' to everything
> > Common Practice so as not to get confused with data from other tables. Admittedly not needed as we are only working with a single table. But there you go.

Good point.